### PR TITLE
feature/admin-tweaks

### PR DESCRIPTION
**Pull request by arthurian**
Modified django admin interface display fields so that it's easier to find specific resources, collections, etc. Also improved the performance of the ORM's sql queries on some screens by adding JOINs where needed.

@elliottyates Review?

---
Note: this PR is not tied to a particular Jira story, but simply categorized as _technical debt_ that was fixed in the course of working on #24 and #25.

### CODE REVIEW

File: media_service/admin.py [hunk 1]
```diff
@@ -11,26 +11,53 @@ class ResourcesInline(admin.StackedInline):
...
 
 class CollectionAdmin(admin.ModelAdmin):
     list_display = ('id', 'title', 'course', 'sort_order')
     ordering = ('course', 'sort_order')
+    search_fields = ('title',)
     inlines = (CollectionResourcesInline,)
 
+    def get_queryset(self, request):
+        qs = super(CollectionAdmin, self).get_queryset(request)
+        return qs.select_related('course')
```
Comment (elliottyates) [inline media_service/admin.py]: Not sure what the interface looks like, but is this something you want to add to `CollectionsInline` as well, like you did with `ResourcesInline`? (And, for that matter, `CollectionResourcesInline`?)

File: media_service/admin.py [hunk 2]
```diff
@@ -11,26 +11,53 @@ class ResourcesInline(admin.StackedInline):
...
+    def get_queryset(self, request):
+        qs = super(UserProfileAdmin, self).get_queryset(request)
+        return qs.select_related('user')
+
 admin.site.register(MediaStore, MediaStoreAdmin)
 admin.site.register(Course, CourseAdmin)
 admin.site.register(Collection, CollectionAdmin)
-admin.site.register(Resource)
+admin.site.register(Resource, ResourceAdmin)
 admin.site.register(CollectionResource)
```
Comment (elliottyates) [inline media_service/admin.py]: If `CollectionResource`s generate multiple SQL statements, they might merit their own admin as well.

**Comment by elliottyates** [comment]
Looks like our #github slack integration swallows the 'approval' part of the new github review flow (and the comment that goes with it). So, 👍

**Comment by arthurian** [comment]
Thank @elliottyates. Yes, I think some of the other registered models might benefit from similar performance tuning. This one just addressed the most egregious/frequently used ones.

**Pull request merged**